Makefile

```
@@ -20,6 +20,8 @@ CATALOG_TEMP := $(shell mktemp -d)
 
 ## Development
 
+all: lint test-unit operator-image bundle-image
```

checkmake (0.3.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)